language/use-cases/document-summarization/summarization_with_documentai.ipynb

```diff
@@ -402,21 +401,30 @@
     "    # Process the document.\n",
     "    document = process_document(processor_name, file_path=path)\n",
     "\n",
-    "    if document:\n",
-    "        # Split the text into chunks of the specified size.\n",
-    "        document_chunks = textwrap.wrap(text=document.text, width=chunk_size)\n",
-    "\n",
-    "        # Loop through each chunk and create a dictionary with metadata and content.\n",
-    "        for chunk_number, chunk_content in enumerate(document_chunks, start=1):\n",
-    "            # Append the chunk information to the extracted_data list.\n",
-    "            extracted_data.append(\n",
-    "                {\n",
-    "                    \"file_name\": file_name,\n",
-    "                    \"file_type\": file_type,\n",
-    "                    \"chunk_number\": chunk_number,\n",
-    "                    \"content\": chunk_content,\n",
-    "                }\n",
-    "            )"
+    "    if not document:\n",
+    "        continue\n",
+    "    \n",
+    "    # Using Document AI Toolbox to handle post-processing\n",
+    "    wrapped_document = Document.from_documentai_document(document)\n",
+    "\n",
+    "    # Split the text into chunks of the specified size based on layout.\n",
```

Comment:
I don't quite understand how this is chunking the content based on this comment. Is it possible to reword/simplify this comment similar to the other examples below that are "chunked by page" or "chunked by paragraph"?

Reply:
Updated